packages/react-components/src/components/FileDownloadCards.tsx

```diff
@@ -244,5 +252,5 @@ const DownloadIconTrampoline = (): JSX.Element => {
 const useLocaleStringsTrampoline = (): _FileDownloadCardsStrings => {
   /* @conditional-compile-remove(file-sharing) */
```

Comment:
add @conditional-compile-remove(teams-inline-images-and-file-sharing) here as well